### PR TITLE
DOC-6559: Updating the violation event attributes doc

### PR DESCRIPTION
**Pull request by paperclypse**
I've added new event attributes. 

I still have some more work on this to do, so I'm making this a draft PR for the time being.

### CODE REVIEW

**Comment by paperclypse** [comment]
I pinged our developer friends to help with these failing checks.

File: src/data-dictionary/events/NrAiIncident/tags.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: tags.*
+type: attribute
+events:
+  - NrAiIncident
+---
+
+Arbitrary key-value metadata, or tags, associated with the violation. `tag.` is the prefix and `*` is the metadata/tag name. For details on how to use this, see the documentation for [muting rules](/docs/alerts/new-relic-alerts/managing-notification-channels/muting-rules-suppress-notifications) or [description](/docs/alerts/new-relic-alerts/defining-conditions/alert-condition-descriptions).
+
```
Comment (urbiz-nr) [inline src/data-dictionary/events/NrAiIncident/tags.md:9]: I'd specify what is the description about, it looks out of context.

**Review by urbiz-nr — DISMISSED** [review]
Wow, great job! Please check my comment, well done in any case!

File: src/data-dictionary/events/NrAiIncident/tags.md (hunk 1 above)
Comment (paperclypse) [reply src/data-dictionary/events/NrAiIncident/tags.md:9]: Good call!

**Pull request merged**